src/integration/pages/mediaAssetPage/user.js

```diff
@@ -0,0 +1,49 @@
+const { within } = require('@testing-library/dom');
+
+const { amp, canonical } = global;
+
+export default ({
+  headlineText,
+  timestamp,
+  bulletedListItem,
+  relatedContentHeadline,
+}) => {
+  [amp, canonical].forEach(page => {
+    describe(`And using ${page.platform}`, () => {
+      it('I can see the headline', () => {
+        const { getByText } = within(
+          page.document.querySelector('h1[id="content"]'),
+        );
+
+        const headline = getByText(headlineText);
+        expect(headline).toBeInTheDocument();
+      });
+
+      it('I can see the timestamp', () => {
+        const { getByText } = within(page.document.querySelector('time'));
+
+        const timestampEl = getByText(timestamp);
+
+        expect(timestampEl).toBeInTheDocument();
+      });
+
+      it.skip('I can see the bulleted list item', () => {
+        const { getAllByText } = within(
+          page.document.querySelector(`[class^="Wrapper"]`),
```

Comment:
Is `within` use here necessary? Also, that's a weak looking selector. I can imagine other elements in the page could have a class of Wrapper. I wouldn't encourage class name query selectors anyway especially in the user tests. A stronger query selector might be something like `main [role="list"]`. Even then you could use one of the DOM Testing Library queries for this sort of thing https://testing-library.com/docs/dom-testing-library/api-queries such as `getByRole` e.g.

```js
const [bulletedListItemEl] = page.getByRole('list')
```

Reply:
This scenario - and the related content one - really need some work.  They both use lists, and sometimes it is not possible to determine which one is which, and that's why I had to revert to looking at the class.  It think we should take a look at these in more detail - once the rest has been merged.